lib/whitehall_importer/edition_history.rb

```diff
@@ -96,6 +96,10 @@ def imported_entry_type(event, edition_number)
       end
     end
 
+    def last_event
```

Comment:
I wonder if we need this method, it fits into the idea that this class should handle all edition history logic but as the only logic is `revision_history.last` which is pretty self explanatory I think it'd be fine to have it inline in the create_edition file which would be a nice code reduction. What do you think?